File: application-2.1/tek-sorumluluk-c#/tekSorumlulukAdres/Adres.cs
public class Adres
{
    public string Ulke { get; set; }
    public string Il { get; set; }
    public string Ilce { get; set; }
    public int PostKodu { get; set; }

    public Adres(string ulke = "", string il = "", string ilce = "", int postKodu = 0)
    {
        Ulke = ulke;
        Il = il;
        Ilce = ilce;
        PostKodu = postKodu;
    }

    public override string ToString()
    {
        return $"{Ulke}, {Il}, {Ilce}, {PostKodu}";
    }
}

File: application-2.1/tek-sorumluluk-python/Adres.py
class Adres:
    def __init__(self, ulke="", il="", ilce="", post_kodu=0):
        self._ulke = ulke
        self._il = il
        self._ilce = ilce
        self._post_kodu = post_kodu

    @property
    def ulke(self):
        return self._ulke

    @ulke.setter
    def ulke(self, ulke):
        self._ulke = ulke

    @property
    def il(self):
        return self._il

    @il.setter
    def il(self, il):
        self._il = il

    @property
    def ilce(self):
        return self._ilce

    @ilce.setter
    def ilce(self, ilce):
        self._ilce = ilce

    @property
    def post_kodu(self):
        return self._post_kodu

    @post_kodu.setter
    def post_kodu(self, post_kodu):
        self._post_kodu = post_kodu

    def __str__(self):
        return f"{self.ulke}, {self.il}, {self.ilce}, {self.post_kodu}"
